internal/text/text.go

```diff
@@ -63,12 +63,12 @@ func FuzzyAgoAbbr(a, b time.Time) string {
 	return b.Format("Jan _2, 2006")
 }
 
-// DisplayURL returns a copy of the string urlStr removing everything except the hostname and path.
+// DisplayURL returns a copy of the string urlStr removing everything except the scheme, hostname, and path.
 // If there is an error parsing urlStr then urlStr is returned without modification.
 func DisplayURL(urlStr string) string {
 	u, err := url.Parse(urlStr)
 	if err != nil {
 		return urlStr
 	}
-	return u.Hostname() + u.Path
+	return u.Scheme + "://" + u.Hostname() + u.Path
```

Comment:
Hi, we intentionally do not want display URLs to be clickable, because we won't want users to click on them. `DisplayURL()` is only called to display a notice that a URL will be opened in the person's web browser. It's meant as a hint, not as a URL-opening mechanism. Notice that query string parameters or URI fragment are omitted from the display URL too.

Reply:
Hmm, unfortunately that directly contradicts my experience. :-/

When I run either of these commands after new permissions are needed (which is almost every day), they fail specifically because `gh cs` thinks it's not in an interactive shell, and never opens the URL for me:

```fish
gh cs code --codespace (gh cs create --repo github/github --devcontainer-path .devcontainer/devcontainer.json)
// or
gh cs create --repo github/github --devcontainer-path .devcontainer/devcontainer.json | xargs gh cs code --codespace
```

Is there particular harm in letting the URLs be clickable, even if they could also directly be opened?

Alternatively, is there a different fix I could submit to fix the UX for my situation?

Reply:
Ah, now we're getting to be crux of the issue, thanks. It looks like in the codespace permissions prompt specifically, DisplayURL was misused https://github.com/cli/cli/blob/8891456b4bd376dabc34b5e4cb9b68ebc9995b23/pkg/cmd/codespace/create.go#L287-L293

DisplayURL should _not_ be used to print clickable URLs to the terminal. instead, a URL should be directly printed verbatim as a string.

You're welcome to rework your PR to change this usage specifically. Thanks!